controllers/progresses.js

```
@@ -267,4 +269,51 @@ const getProgressBydDateController = async (req, res) => {
+        failedRecords: result.failedRecords
+      }
+    });
+  } catch (error) {
+    logger.error(`Error in bulk progress creation: ${error.message}`);
+    return res.status(500).json({
+      message: INTERNAL_SERVER_ERROR_MESSAGE,
+    });
+  }
+};
+
+module.exports = { 
```

ESLint

[error] Trailing spaces not allowed.

(no-trailing-spaces)